HTTPTask.swift

```diff
@@ -520,6 +545,15 @@ public class HTTPTask : NSObject, NSURLSessionDelegate, NSURLSessionTaskDelegate
         }
     }
     
+    public func URLSession(session: NSURLSession, task: NSURLSessionTask, willPerformHTTPRedirection response: NSHTTPURLResponse, newRequest request: NSURLRequest, completionHandler: (NSURLRequest!) -> Void) {
+    
+        if let reDirect = redirection {
+            reDirect(session: session, task: task, response: response, request: request, completionHandler: completionHandler)
+        }
+
```

Comment:
Looking at the docs, do we need to pass nil to the completionHandler in order for it to do the default logic when no redirect closure is provided?